contracts/utils/Address.sol

```diff
@@ -76,26 +80,50 @@ library Address {
         if (address(this).balance < value) {
             revert Errors.InsufficientBalance(address(this).balance, value);
         }
-        (bool success, bytes memory returndata) = target.call{value: value}(data);
-        return verifyCallResultFromTarget(target, success, returndata);
+        bool success = LowLevelCall.callNoReturn(target, value, data);
+        if (success && (LowLevelCall.returnDataSize() > 0 || target.code.length > 0)) {
+            return LowLevelCall.returnData();
+        } else if (success) {
+            revert AddressEmptyCode(target);
+        } else if (LowLevelCall.returnDataSize() > 0) {
+            LowLevelCall.bubbleRevert();
+        } else {
+            revert Errors.FailedCall();
+        }
```

Comment:
This saves a few more gas compared to [your numbers here](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/5094#issuecomment-3072461643).
Same pattern in `functionCallWithValue(..)` & `functionStaticCall(..)` saves gas.
```suggestion
        if (LowLevelCall.callNoReturn(target, value, data)) {
            if (LowLevelCall.returnDataSize() > 0 || target.code.length > 0) {
                return LowLevelCall.returnData();
            }
            revert AddressEmptyCode(target);
        }
        if (LowLevelCall.returnDataSize() > 0) {
            LowLevelCall.bubbleRevert();
        } else {
            revert Errors.FailedCall();
        }
```